app/build.gradle.kts

```
@@ -87,12 +87,16 @@ android {
     checkDependencies = true
   }
 
+  buildFeatures {
+    buildConfig = true
+  }
+
   buildTypes {
-    getByName("release") {
+    release {
       Config.ANDROIDX_MEDIA_EXTENSIONS.forEach { extension ->
         val proguardFile = file(
           "../thirdparty/androidx-media/libraries/${extension}/proguard-rules.txt"
-        )
+        }
```

```suggestion
        )
```
Made a typo while resolving a merge conflict, my apologies for any rudimentary notifications and a little mess in the commits list caused.